extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -99,29 +99,7 @@
 	'communitypage-todo-module-heading' => 'Community to-do list',
 	'communitypage-todo-module-edit-list' => 'Edit list',
 	'communitypage-todo-module-description' => 'This is an area for admins to suggest pages that need work to help new contributors get started. (Admins - remember to provide details on what needs to be done.)',
-	'communitypage-todo-module-zero-state' => 'Want to know how you can contribute to this community? Contact an admin and ask them to provide suggestions here.',
-	// TODO: to be removed in the scope of: https://wikia-inc.atlassian.net/browse/WW-83
-	'communitypage-popularpages-mediumtitle' => 'Pages that need proofreading',
-	'communitypage-popularpages-hightitle' => 'Please proofread these pages',
-	'communitypage-popularpages-mediumdescription' => 'Please proofread these pages and fix typos, grammatical mistakes or factual errors.',
-	'communitypage-popularpages-highdescription' => 'Make sure there are no typos, grammatical mistakes or factual errors.',
-	'communitypage-popularpages-edittext' => 'Read and fix',
-	'communitypage-uncategorizedpages-mediumtitle' => 'Pages that need categories',
-	'communitypage-uncategorizedpages-hightitle' => 'Please add categories to these pages',
-	'communitypage-uncategorizedpages-mediumdescription' => 'Help us organize this wiki by adding categories to these pages!',
-	'communitypage-uncategorizedpages-highdescription' => 'These pages currently have zero categories and need one or more added. Examples of categories include “Characters” or “Items” or “Locations.”',
-	'communitypage-uncategorizedpages-edittext' => 'Add categories',
-	'communitypage-wantedpages-mediumtitle' => 'Pages that need to be created',
-	'communitypage-wantedpages-hightitle' => 'Please create these pages',
-	'communitypage-wantedpages-mediumdescription' => 'There are links pointing to these pages, but they don\'t exist! Please create them. Write a few sentences to get it started!',
-	'communitypage-wantedpages-highdescription' => 'Write a few sentences to get it started.',
-	'communitypage-wantedpages-edittext' => 'Create page',
-	'communitypage-deadendpages-mediumtitle' => 'Pages that need links',
-	'communitypage-deadendpages-hightitle' => 'Please add links to these pages',
-	'communitypage-deadendpages-mediumdescription' => 'Please add links to these pages. Well-linked articles help readers discover more information about your topic and improve search engine rankings.',
-	'communitypage-deadendpages-highdescription' => 'Add links to any important names or words that appear in the first few sentences on these pages.',
-	'communitypage-deadendpages-edittext' => 'Add links',
-	'right-first-edit-dialog-exempt' => 'Not shown a welcome dialog upon their first edit',
+	'communitypage-todo-module-zero-state' => 'Want to know how you can contribute to this community? Contact an admin and ask them to provide suggestions here.'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-todo-module-zero-state" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)